test/potluck_test.rb

```diff
@@ -0,0 +1,69 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/dish'
+require './lib/potluck'
+
+class PotluckTest < Minitest::Test
+
+  def test_it_exists
+    potluck = Potluck.new("7-13-18")
+
+    assert_instance_of Potluck, potluck
+  end
+
+  def test_it_has_a_date
+    potluck = Potluck.new("7-13-18")
+
+    assert_equal "7-13-18", potluck.date
+  end
+
+  def test_it_is_an_empty_array
+    potluck = Potluck.new("7-13-18")
+
+    assert_equal [], potluck.dishes
+  end
+
+  def test_it_exists_couscous
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+
+    assert_instance_of Dish, couscous_salad
+  end
+
+  def test_it_exists_cocktail
+    cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+
+    assert_instance_of Dish, cocktail_meatballs
+  end
+
+  def test_it_can_add_couscous_salad
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    potluck = Potluck.new("7-13-18")
+
+    assert_equal [], potluck.dishes
+    potluck.add_dish(couscous_salad)
+
+    assert_equal [couscous_salad], potluck.dishes
+  end
+
+  def test_it_can_add_cocktail_meatballs
+    cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+    potluck = Potluck.new("7-13-18")
+
+    assert_equal [], potluck.dishes
+    potluck.add_dish(cocktail_meatballs)
+
+    assert_equal [cocktail_meatballs], potluck.dishes
+  end
+
+  def test_it_has_a_dishes_length_of_2
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+    potluck = Potluck.new("7-13-18")
+
+    assert_equal [], potluck.dishes
+    potluck.add_dish(couscous_salad)
+    potluck.add_dish(cocktail_meatballs)
+
+    assert_equal 2, potluck.dishes.length
```

Comment:
ooh! haha, i'm reading these tests one at a time and now that I've gotten to the bottom, you did almost exactly what i recommended above, which is great.  So this is a really good thought; testing `length` though is not a good test, because just because something is the right length doesn't mean that it's a collection of the right stuff (if potluck.dishes was an array of strings, this test would still pass, even though it should actually be an array of dish objects), so it can give you false positives. 

So a better assertion would be similar to what I suggested above: `assert_equal [couscous_salad, cocktail_meatballs], potluck.dishes`

That assertion covers that it should have a length of 2 (because you are specifically asserting two things in the array), but also that they are specifically the couscous_salad dish object and the cocktail_meatballs dish object, not just two of something random.